physionet-django/project/managers/publishedproject.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models import Manager, Q
+from project.models import AccessPolicy, DUASignature, DataAccessRequest
+from user.models import Training, TrainingType
+
+
+class PublishedProjectManager(Manager):
+    def accessible_by(self, user):
```

Comment:
We need a way to fetch all the projects that a specified user has access to (part of the contract required by the `hdn-research-environment` package). There is the [has_access](https://github.com/MIT-LCP/physionet-build/blob/dev/physionet-django/project/modelcomponents/publishedproject.py#L206) method available to instances of `PublishedProject`, but using it will be very inefficient (N+1 queries).

The `accessible_by` method only uses a few queries to fetch this information.